src/ui/component/fileRender/view.jsx

```diff
@@ -155,9 +156,24 @@ class FileRender extends React.PureComponent<Props> {
     }
     // @endif
 
-    // Message Error
-    const unsupportedMessage = __("Sorry, we can't preview this file.");
-    const unsupported = <LoadingScreen status={unsupportedMessage} spinner={false} />;
+    const unsupported = IS_WEB ? (
+      <div className={'content__cover--disabled'}>
+        <Yrbl
+          className={'content__cover--disabled'}
+          title={'Not available on LBRY TV'}
+          subtitle={'You can view or download this file by installing the App'}
```

Comment:
`You can view this content in the LBRY app. <Button label={"Download it here"} href="lbry.com/get"...`